products/views.py

```diff
@@ -1,3 +1,48 @@
-from django.shortcuts import render
+import json
+from django.http     import JsonResponse, request
+from django.views    import View
+from products.models import Product, Image, Category
+from users.models    import User
 
-# Create your views here.
+class HostEnrollment(View):
+    #@login_decorator
+    def post(self,request):
+        data = json.loads(request.body)
+        print(request)
+        user = request.user
+
+        if data["name"] == "" or data["head_count"] == "" or data["price"] == "":
+          return JsonResponse ({"MESSAGE": "REQUIRE_INFORM_NULL"}, status = 400)
+
+        MAX_PEOPLE = 4
+        if data["head_count"] >= MAX_PEOPLE :
+            return JsonResponse ({"MESSAGE":"PEOPLE_INPUT_ERROR"})   
+
+        product = Product.objects.create(
+            user_id        = request.user, 
+            name           = data["name"],
+            head_count     = data["head_count"],
+            price          = data["price"],
+            description    = data["description"],
+            address        = data["address"],
+            detail_address = data["detail_adress"]
+          )
+
+        for image_id in data["image_id"]:
+            Image.objects.create(product = product, image = data["image"])
+
+        addressSplit = data["product.address"].split(" ")[0] 
+        Category.objects.create(
+            product       = product,
+            big_address   = data["product.address"].split(" ")[0],
+            small_address = addressSplit[1],
+        )
+        
```

Comment:
`addressSplit = data["product.address"].split(" ")[0]`을 직접 선언하지 않고, 아래와 같이 사용하는게 좋을 것 같네요~
왜 그럴까요??

```suggestion
        Category.objects.create(
            product       = product,
            big_address   = data["product.address"].split(" ")[0],
            small_address = data["product.address"].split(" ")[1],
        )
        
```

Reply:
변수를 선언하지 않고 코드 안에서 해결하는 것이 더욱 간결하고, 코드를 통일된 방식으로 작성할 수 있기 때문인 듯 합니다..!